### DISCUSSION TITLE
HLS Dependency issue

### DISCUSSION THREAD

**Discussion by ashkanyadegari** [vidstack/player#1183] [Q&A] [answered]
Hi all,

I am based in China and I love using vidstack. However, it seems like that the HLS dependency is still dependent on getting it from JSDeliver CDN. Does anyone know how I can change this to have HLS directly installed in my project.

This is my package.json:
```
    "@stitches/react": "^1.2.8",
    "@tanstack/react-table": "^8.10.7",
    "@vidstack/react": "^1.8.0",
    "axios": "^1.6.2",
    "blaze-slider": "^1.9.3",
    "class-variance-authority": "^0.7.0",
    "clsx": "^2.0.0",
    "cmdk": "^0.2.0",
    "date-fns": "^2.30.0",
    "dayjs": "^1.11.10",
    "embla-carousel-react": "^8.0.0-rc15",
    "framer-motion": "^10.16.12",
    "hls.js": "^1.4.12",
    "js-cookie": "^3.0.5",
    "keen-slider": "^6.8.6",
    "lodash.debounce": "^4.0.8",
``` 

However, when I check my console in my web application. I get the following error:

```
cdn.jsdelivr.net/npm/hls.js@%5E1.0.0/dist/hls.min.js:1 
        
 Failed to load resource: net::ERR_NETWORK_CHANGED
 ```
 
 I want to make sure that HLS is directly used from the npm package I have installed.

**Comment by mihar-22** [comment, marked as answer]
Hey @ashkanyadegari! The "importing" section in our [HLS provider docs](https://www.vidstack.io/docs/player/api/providers/hls?styling=default-theme#importing) shows you how to load `hls.js` locally.

**Reply by ashkanyadegari** [reply]
Wow! So stupid of me to have overseen that. Thank you so much!
